pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -174,6 +174,7 @@ self: super: {
   marquise = dontCheck super.marquise;                  # https://github.com/anchor/marquise/issues/69
   memcached-binary = dontCheck super.memcached-binary;
   msgpack-rpc = dontCheck super.msgpack-rpc;
+  persistent-migration = dontCheck super.persistent-migration;
```

Comment:
Can you move this into `configuration-nix.nix` and add a comment stating something along the lines of “Test suite needs a postgresql database”?

`configuration-common.nix` is used for workarounds that are expected to be temporary and `configuration-nix.nix` for things that we'll have to always do (because of the way the package / Nix work), as a rule of thumb

Reply:
done. thanks for the explanation.